.gitignore

```diff
@@ -30,10 +30,6 @@ dependency-reduced-pom.xml
 
 gen-external-apklibs
 
-bazel-bin
-bazel-genfiles
-bazel-out
-bazel-testlogs
-bazel-dagger
+bazel-*
```

Comment:
`bazel-dagger` is actually `bazel-WORKSPACENAME` (I had the repo checked out as dagger2 => bazel-dagger2).

Reply:
Better to be explicit

Reply:
No. The generated folder is named "bazel-WORKSPACENAME" where WORKSPACENAME is the name of the folder in which you're put the dagger project. If you check out dagger in a "foobar" folder, a "bazel-foobar" folder will be created.

Since there's no checking in "dagger-*" file anyway, using this pattern works better because it works for all.